specification/maps/data-plane/Search/preview/2022-09-01-preview/search.json

```diff
@@ -0,0 +1,966 @@
+      "type": "object",
+      "properties": {
+        "batchItems": {
+          "description": "The list of queries to process.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/GeocodingBatchRequestItem"
+          }
+        }
+      }
+    },
+    "GeocodingBatchResponseItem": {
+      "type": "object",
+      "properties": {
+        "optionalId": {
+          "description": "id of the batchItem which would be the same as the id in the request",
+          "type": "string"
+        },
+        "type": {
+          "$ref": "#/definitions/FeatureType"
+        },
+        "features": {
+          "$ref": "#/definitions/Features"
+        },
+        "nextLink": {
+          "$ref": "#/definitions/NextLink"
+        },
+        "error": {
+          "$ref": "../../../../../common-types/data-plane/v1/types.json#/definitions/ErrorDetail"
+        }
+      }
+    },
+    "GeocodingBatchResponse": {
+      "description": "This object is returned from a successful Geocoding Batch service call.",
+      "type": "object",
+      "properties": {
+        "summary": {
+          "description": "Summary for the batch request",
+          "type": "object",
+          "readOnly": true,
+          "properties": {
+            "successfulRequests": {
+              "description": "Number of successful requests in the batch",
+              "type": "integer",
+              "format": "int32",
+              "readOnly": true
+            },
+            "totalRequests": {
+              "description": "Total number of requests in the batch",
+              "type": "integer",
+              "format": "int32",
+              "readOnly": true
+            }
+          }
+        },
+        "batchItems": {
+          "description": "Array containing the batch results.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/GeocodingBatchResponseItem"
+          }
+        }
+      }
+    },
+    "Features": {
+      "type": "array",
+      "items": {
+        "$ref": "#/definitions/FeaturesItem"
+      }
+    },
+    "FeaturesItem": {
+      "type": "object",
+      "properties": {
+        "type": {
+          "type": "string",
+          "description": "must be Feature",
+          "enum": [
+            "Feature"
+          ],
+          "x-ms-enum": {
+            "name": "FeaturesItemTypeEnum",
+            "modelAsString": true,
+            "values": [
+              {
+                "value": "Feature"
+              }
+            ]
+          }
+        },
+        "id": {
+          "type": "string",
+          "description": "ID for feature returned"
+        },
+        "properties": {
+          "x-ms-client-flatten": true,
+          "type": "object",
+          "properties": {
+            "type": {
+              "type": "string",
+              "description": "One of: \n* Address\n* RoadBlock\n* RoadIntersection",
+              "readOnly": true
```

Comment:
Recommend to remove `readOnly: true` if this schema is not used in a request-body.